deploy/fabfile.py

```diff
@@ -33,26 +32,58 @@
 home = "/home/site"
 log = "%s/dashboard.log" % home
 current = "%s/%s/dashboard" % (home, environment)
+now = time.strftime("%Y-%m-%d", time.localtime())
 
 # principal command to run upon update
-command = "cd %s && git pull && git submodule update && jekyll build >> %s" % (current, log)
+staging = "cd %s && git pull && git submodule update --remote && ./_data/import-public.rb && git add _data/projects.yml && git commit -m 'Update data for %s' && git push && ./go build >> %s" % (current, now, log)
```

Comment:
Cool. I've added the Gemfile.lock to the repo. This doesn't feel like the kind of thing the machine user should update, but something we should manage in development environments and push up later. That's [based on my understanding of how bundle works](http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/) but I'd appreciate any clarity. cc: @adelevie